src/pages/ProductList/ProductList.jsx

```diff
@@ -1,7 +1,153 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import rArrow from '../../assets/images/right-arrow.png';
+import lArrow from '../../assets/images/left-arrow.png';
 
 const ProductList = () => {
-  return <div>ProductList</div>;
+  const [reservation, setReservation] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productList.json')
+      .then(result => result.json())
+      .then(data => setReservation(data));
+  }, []);
+  // console.log(reservation);
+
+  return (
+    <ProductListWrapper>
+      <CateBox>
+        <CateTitle>원피스제품</CateTitle>
+        <CateSortBox>
+          {CATEGORY_SORT.map(item => (
+            <CateSort to="/" key={item.id}>
+              {item.title}
+            </CateSort>
+          ))}
+        </CateSortBox>
+      </CateBox>
+      <ProductBox>
+        {reservation.map(product => (
+          <ProductCard key={product.id}>
+            <ProductImage src={`../images/${product.image}`} />
+            <ProductTitle>{product.title}</ProductTitle>
+            {product.price}
+          </ProductCard>
+        ))}
+      </ProductBox>
+      <PageNationBox>
+        <PageNationList>
+          <PageLNarrow to="/" />
+          <PageNationLink to="/">1</PageNationLink>
+          <PageNationLink to="/">2</PageNationLink>
+          <PageNationLink to="/">3</PageNationLink>
+          <PageNationLink to="/">4</PageNationLink>
+          <PageNationLink to="/">5</PageNationLink>
+          <PageRNarrow to="/" />
+        </PageNationList>
+      </PageNationBox>
+    </ProductListWrapper>
+  );
 };
+// src="../images/left-arrow.png"
+const ProductListWrapper = styled.div`
+  width: 1020px;
+  margin: 0 auto;
+  padding: 10px;
+`;
+const CateBox = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const CateTitle = styled.p`
+  padding: 10px;
```

Comment:
```suggestion
const CateBox = styled.div`
  display: flex;
  flex-direction: column;
`;

const CateTitle = styled.p`
  padding: 10px;
```

스타일끼리 가독성을 위해 한칸씩 띄워주세요!